docs/Building.md

```diff
@@ -0,0 +1,73 @@
+
+# Building
+
+This interface utilizes **Tauri** and requires ***[some extra setup]***.
+
+<br>
+
+## Sidecar
+
+The interface uses **Sidecar** as it's backend for communication <br>
+with **OpenVR** and must be built first for the project to run.
+
+1. Navigate to `sidecar/`.
+
+2. **Create** & **Enter** a folder named `build`.
+
+3. Run CMake with:
+
+    ```sh
+    cmake ..
+    ```
+
+4. Open the generated **Visual Studio** project.
+
+5. **Build** the project in a `Release` / `MinSizeRel` mode.
+
+6. Copy the build **Artifacts** into `src-tauri`.
+    - `sidecar-x86_64-pc-windows-msvc.exe`
+    - `openvr_api.dll`
+
+<br>
+
+## Interface
+
+1. Have **Sidecar** built.
```

Comment:
Would be nice to have `**Sidecar**` link to `...main/docs/Building.md#sidecar` so that it bring people to that part of the page.